index.html

```diff
@@ -0,0 +1,245 @@
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_03.png" alt="program icon" class="img-fluid mb-4">
+                            <p class="card-subtitle">Forum</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_04.png" alt="Program icon" class="img-fluid mb-4">
+                            <p class="card-subtitle">Workshop</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Try creating your own work using open source license rather than just watching at it.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_05.png" alt="Program 5 icon" class="img-fluid mb-4">
+                            <p class="card-subtitle">CC Ignite</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">get opportunities to with CC affiliates around the world, also after the summit.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="row pb-5 align-items-center text-center justify-content-center">
+                <div class="d-none d-md-block see-whole">
+                    SEE THE WHOLE PROGRAM
+                </div> 
+                <div class="d-block d-md-none join-summit">
+                    Join CC Global Summit 2023
+                </div> 
+            </div>
+        </div>
+    </section>
+
+    <!-- feature speakers -->
+    <section id="speakers">
+        <div class="text-center">
+            <h2 class="speakers">Featured Speakers</h2>
+        </div>
+        <div class="container-lg pb-3">
+            <div class="row align-items-center justify-content-center pt-5" id="mainSection">
+
+              <!-- speakers dynamic code  -->
+
+            </div>
+            <div class="col-md-4 more-btn text-center">
+                MORE <i class="bi bi-chevron-down"></i>
+            </div>
+        </div>
+      
+    </section>
```

Comment:
- [ ]  As required for the project to stick as much as possible to the one form the Behance template, kindly ensure that you improve again the design of the above section as you miss some features to improve such as the `grid backgrounf image` on each speaker image, font(size, weight,,,), on the section title, spacing around it and the horizontal button below it, text color,....
### Template design
![Screenshot from 2023-07-06 19-14-53](https://github.com/MozamelJawad/Capstone-Project/assets/83042742/5b6a4dc8-55e1-469d-b072-9aad8e4a6a70)
### Your Dsign
![Screenshot from 2023-07-06 19-16-12](https://github.com/MozamelJawad/Capstone-Project/assets/83042742/2c1216f4-3971-49db-9969-cc3063f002d3)